src/bootstrap_handler.rs

```diff
@@ -184,6 +194,33 @@ mod test {
         path.parent().unwrap().join(name)
     }
 
+    fn create_contacts() -> Contacts {
+        let addr = net::SocketAddrV4::new(net::Ipv4Addr::new(1,2,3,4), 8080);
```

Comment:
Here is something that may help `use net::test::{next_test_ip4, next_test_ip6};`